src/status_im/contexts/chat/messenger/messages/navigation/view.cljs

```diff
@@ -132,12 +132,11 @@
                    [all-loaded-sub])
     [rn/view
      {:style (style/navigation-view navigation-view-height messages.constants/pinned-banner-height)}
-     (when (seq last-message)
```

Comment:
don't know why we added this

Reply:
From what I can remember, It was a fix for [this](https://github.com/status-im/status-mobile/pull/19459#issuecomment-2044480588)

Reply:
thank you @ibrkhalil for adding context. I am still confused, how last-message relates to shadow :thinking:

Reply:
Sorry for not being specific, It's the overlapping of the top nav bar with the avatar.

Reply:
Thank you. Unfortunately, I am not able to reproduce this overlap, if this ever comes up let's log an issue perhaps with a video to give more context.

Also, I don't think the last message is a proper fix if the top bar overlaps. We should not just remove the top bar when there is no last message. If it overlaps then probably something wrong with the animation, as you can see from the shared image position of an avatar with respect to the close button is wrong.


![image](https://github.com/status-im/status-mobile/assets/17097240/b7e413ae-29a4-4779-8a62-9968ec68bb77)

Reply:
I am requesting design review as it was approved on my PR.